### PR TITLE
zib-Refraction

### PR DESCRIPTION
**Pull request by jduwel**

### CODE REVIEW

File: resources/zib/ext-AnatomicalLocation.Laterality.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pieter-edelman-nictiz) [inline resources/zib/ext-AnatomicalLocation.Laterality.xml:1]: Shouldn't be included in the pull request

File: resources/zib/zib-AnatomicalLocation.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pieter-edelman-nictiz) [inline resources/zib/zib-AnatomicalLocation.xml:1]: Shouldn't be included in the pull request

File: resources/nl-core/nl-core-AnatomicalLocation.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (pieter-edelman-nictiz) [inline resources/nl-core/nl-core-AnatomicalLocation.xml:1]: Shouldn't be included in the pull request.

**Review by pieter-edelman-nictiz — CHANGES_REQUESTED** [review]
The AnatomicalLocation materials shouldn't be part of the pull request. Other than that, everything looks fine!

**Comment by jduwel** [comment]
1 remarkt: both Observation.code and Observation.component:readAddition.code have a mapping on SNOMED code 251718005 - https://bits.nictiz.nl/browse/ZIB-1443. Maybe one of the codes in the profile should be removed until this is resolved?

**Comment by ArdonToonstra** [comment]
Made some changes to this PR. I think it is ok now!
We do have the remark stated by Jorn which we should keep track of. I changed the code in `.component.code` from 251718005 to TODO-ZIB-1443.

**Review by LuudSlagter — APPROVED** [review]
Laatste wijzigingen van Niek gecheckt en goedgekeurd (en nog een kleine aanvulling gedaan).

**Review by pieter-edelman-nictiz — APPROVED** [review]

**Pull request merged**